.travis.yml

```diff
@@ -0,0 +1,49 @@
+language: csharp
+mono: none
+dotnet: 2.1.2
+sudo: required
+services:
+  - mysql
+  - postgresql
+  - docker
+env:
+  - DB=SqlServer2008 CONNECTION_STRING="Server=localhost;initial catalog=nhibernate;User Id=sa;Password=P@ssw0rd;"
+  - DB=PostgreSQL CONNECTION_STRING="Host=localhost;Port=5432;Username=postgres;Database=nhibernate;Enlist=true;"
+  - DB=Firebird 
+  - DB=MySQL CONNECTION_STRING="Server=127.0.0.1;Uid=root;Database=nhibernate;Old Guids=True;"   
+matrix:
+  allow_failures:
+    - env: DB=MySQL CONNECTION_STRING="Server=127.0.0.1;Uid=root;Database=nhibernate;Old Guids=True;"
```

Comment:
This bug: https://bugs.mysql.com/?id=89639&edit=2

Reply:
> It should ignore only known failing tests.

I don’t have time nor desire to port the test result comparator at the moment. We would need to extend it to support different tests results for different platform-framework combination. I think it’s fair complex task and needs to be done (if at all) in a different pull request.

Reply:
The result comparer should not be used again ideally. I was thinking of additional ignore rules for those failing test. Something like `if !NETFX \n if (MySql) ignore` could do it, but that is maybe not very satisfactory since it seems more related to MySql data driver than to the fact of not running under NETFX.

So well, ok for letting this as is for now.